x.payment_batching/payment_batching.md

```diff
@@ -0,0 +1,258 @@
+
+    [X] Free withdrawal (payment sent within 6 hours)
+    [ ] Immediate withdrawal (withdrawal fee 0.123 mBTC)
+
+### Reduced privacy
+
+A second concern with payment batching is that it can make users feel
+like they have less privacy.  Every user you pay in the same transaction
+can reasonably assume that everyone else receiving an output from that
+transaction is being paid by you.  If you had sent separate
+transactions, any onchain relationship between the payments might be
+less apparent or even non-existent.
+
+![Screenshot of a possible transaction batch in a block explorer](img/batch-screenshot.png)
+
+Note that transactions belonging to particular Bitcoin services are
+often identifiable by experts even if they don't use payment
+batching, so batching doesn't necessarily cause a reduction in privacy
+for those cases.
+
+It may be possible to partially mitigate this problem by sending batched
+payments in a coinjoin transaction created with other users.  Depending
+on the technique used, this would not necessarily reduce the efficiency
+of batching and could provide significantly enhanced privacy.  However,
+naive implementations of coinjoin previously provided by Bitcoin
+services have had [flaws][coinjoin sudoku] that prevented them from
+providing significant privacy advantages.  As of February 2019, no
+currently-available coinjoin implementation is fully compatible with the
+needs of payment batching.
+
+### Possible inability to fee bump
+
+A final concern is that you may not be able to fee bump a batched
+payment.  Transaction relay nodes such as Bitcoin Core impose limits on
+the transactions they relay to prevent attackers from wasting bandwidth,
+CPU, and other node resources.  By yourself, you can easily avoid
+reaching these limits, but the receivers of the payments you send can
+respend that money in child transactions that become part of the
+transaction group containing your transaction.
+
+The closer to a limit a transaction group becomes, the less likely
+you'll be able to fee bump your transaction using either
+Child-Pays-for-Parent (CPFP) fee bumping or Replace-by-Fee (RBF) fee
+bumping.  In addition, the more unconfirmed children a transaction has,
+the more RBF fee bumping will cost as you'll have to pay for both the
+increased feerate of your transaction as well as for all the potential
+fees lost to miners when they remove any child transactions in order
+to accept your replacement.
+
+Note that these problems are not unique to batched payments---independent
+payments can have the same problem.  However, if an independent payment
+can't be fee bumped because the independent receiver spent their output,
+only that user is affected.  But if a single receiver of a batched
+payment spends their output to the point where fee bumping becomes
+impossible, all the other receivers of that transaction are also affected.
+
+As of Bitcoin Core 0.18 (April 2019), the limits are[^limits] that a
```

Comment:
This notation isn't part of [github flavored markdown](https://github.github.com/gfm/) and so doesn't work if I view the file in github (eg https://github.com/bitcoinops/scaling-book/blob/115d56423916b7e633237999fe7e4f495f4adaca/x.payment_batching/payment_batching.md).

This brings up a larger question of how we're planning to publish this book. Should it be part of the jekyll bitcoinops site, or somewhere different?

Reply:
I'm changing this to an unlinked superscript ¹ for the sake of this PR.

I think making the draft book part of the site would be ideal as we already have tools there for linting and having people linking to it on our site improves pagerank (for what that's worth).  When we've got all the content in place, we can add some build dependencies that will allow compiling the "final" book to PDF, ePub, and other formats if we want.